src/main/java/racingcar/ResultView.java

```diff
@@ -0,0 +1,24 @@
+package racingcar;
+
+public class ResultView {
+    private static final String START_RESULT_MESSAGE = "실행 결과";
+
+    public static void drawResult(Cars cars){
+        cars.stream().forEach(car -> {
+            System.out.println(drawEachCar(car.getStatusLine()));
+        });
+        System.out.println("");
+    }
+
+    private static String drawEachCar(int statusLine) {
+        StringBuilder sb = new StringBuilder();
+        for(int i=0; i<statusLine; i++){
```

Comment:
```suggestion
        for (int i = 0; i < statusLine; i++) {
```

IDE를 사용하신다면 아래와 같이 Code formatting 기능을 사용해주시면 좋을 것 같아요.

https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html